UnitTests/Controllers/EventControllerShould.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Threading.Tasks;
+using Moq;
+using source.Queries;
+using Xunit;
+using System.Collections.Generic;
+using source.Controllers;
+using Microsoft.AspNetCore.Mvc;
+using source.Models;
+using System;
+using source.Database;
+
+namespace UnitTests.Controllers
+{
+    public class EventControllerShould
+    {
+        // System Under Test
+        readonly EventController _evntController;
+        readonly Mock<IEventQuery> __eventDaoMock;
+        readonly Mock<Event> _eventMock;
+
+
+        public EventControllerShould()
+        {
+            __eventDaoMock = new Mock<IEventQuery>();
+
+            _evntController = new EventController(__eventDaoMock.Object);
+        }
+
+        [Fact]
+        public void GetAllEventsByUser_ReturnEvents()
+        {
+
+            //arrange
+            var evt = new Event { eventId = 1, organizerId = "jss94", description= "event description!" };
+            var evts = new List<Event> { evt, evt, evt };
```

Comment:
Careful here is you set your pk to not be duplicated

Reply:
I believe I addressed this as each event now has a unique event id